service/src/loader.rs

```diff
@@ -11,7 +11,7 @@ use cargo::ops::{compile, CompileOptions};
 use cargo::util::homedir;
 use cargo::Config;
 use libloading::{Library, Symbol};
-use log::trace;
+use tracing::log::trace;
```

Comment:
I can't find a `tracing::log` module in the docs. I can find a `tracing::trace` though. Maybe I'm just looking in the wrong place?

Reply:
It's hidden behind the "log" feature on tracing.  The difference is somewhat confusing enough I ended up in the Tokio discord while doing this same thing last time.  The `tracing::trace!` version accumulates the logs internally unless you attach some sort of subscriber (I'm using tracing_subscriber as a dev dependency in some of the tests).  Enabling the "log" feature unveils the `tracing::log::` module where the macros behave as a drop-in replacement for the normal type of logging